File: cpp/Leetcode/Python/problem34.py
class Solution:
    def searchRange(self, nums, target: int):
        lst=[]
        for i in range(len(nums)):
            if nums[i]==target:
                lst.append(i)
        if len(lst)==0:
            return [-1,-1]
        else:
            return [lst[0],lst[-1]]

File: cpp/Leetcode/c++/problem34.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int>lst;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==target){
                lst.push_back(i);
            }
        }
        if(lst.size()==0){
            return {-1,-1};
        }else{
            return {lst[0],lst[lst.size()-1]};
        }
    }
};